bom/pom.xml

```diff
@@ -132,6 +135,11 @@
                 <artifactId>quarkus-amazon-cognito-user-pools-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-s3</artifactId>
+                <version>${aws-java-sdk-s3.version}</version>
```

Comment:
Aren't you mixing v1 and v2 of the SDK by doing so? Is there a reason why you used the v1 one for this?

Reply:
Testcontainers has a dependency on it.